contracts/LockedNORI.sol

```diff
@@ -600,6 +600,8 @@ contract LockedNORI is
     uint256 atTime,
     uint256 amount
   ) internal {
+    require((atTime == 0 && amount > 0) || (atTime > 0 && amount == 0),
```

Comment:
did this not get deployed to mumbai?

Reply:
No, I don't think so.  I guess I didn't run an upgrade since adding this extra check.  We can run a new upgrade today.

Reply:
I just upgrade these contracts on mumbai